multipledispatch/tests/test_benchmark.py

```diff
@@ -0,0 +1,45 @@
+from multipledispatch import dispatch
+import pytest
+
+
+@dispatch(int)
+def isint(x):
+    return True
+
+
+@dispatch(object)
+def isint(x):
+    return False
+
+
+@dispatch(object, object)
+def isint(x, y):
+    return False
+
+
+@pytest.mark.parametrize("val", [1, 'a'])
+def test_benchmark_call_single_dispatch(benchmark, val):
+    benchmark(isint, val)
+
+
+@pytest.mark.parametrize("val", [(1, 4)])
```

Comment:
why is the parametrized over a single call?

Reply:
There are actually two arguments here.  1 and 'a'

Reply:
Mostly so that if we add more complex examples we can see which kinds of types are slower for tests.  You can only use a benchmark once per test for some reason

Reply:
Okay, so it is because you plan on adding more cases?

Reply:
Yeah a large part of the reason I want to do this is for the fancier pytypes based checks in #69 where can dispatch on typed containers.  That will potentially be slow.